### DISCUSSION TITLE
The documentation site 'Edit On GitHub' links return a 404

### DISCUSSION THREAD

**Discussion by benhopkinstech** [octokit/octokit.net#2859] [General]
Hello all,

Sorry if this is the wrong means of doing this but I don't feel it warranted an issue being created due to it not being a problem with the code itself.

When I was browsing the [documentation website](https://octokitnet.readthedocs.io/en/latest/) I noticed that the 'Edit on GitHub' links all seem to return a 404.

The links seem to point to the repository on the `master` branch which must have existed previously before being renamed to `main`.

For example [https://github.com/octokit/octokit.net/edit/master/docs/index.md](https://github.com/octokit/octokit.net/edit/master/docs/index.md) is the link provided when trying to edit the home page. 

This should read [https://github.com/octokit/octokit.net/edit/main/docs/index.md](https://github.com/octokit/octokit.net/edit/main/docs/index.md).

Hopefully it should just be a matter of updating the base url on the documentation site to https://github.com/octokit/octokit.net/edit/main/docs/.

Ben

**Comment by kfcampbell** [comment]
What's really curious to me is that in most cases, such as clicking on the "Getting Started" guide [here](https://octokitnet.readthedocs.io/en/latest/), GitHub will redirect to the correct branch and show a little banner: 

![image](https://github.com/octokit/octokit.net/assets/9327688/3f4affd3-f483-4b7d-8590-be306b54d396)

As for actually fixing this, I'm struggling to find where to make the change. I'm not seeing any relevant configuration in our docs configuration in `mkdocs.yml`. Searching for "master" in our codebase doesn't lead anywhere that looks useful (though incidentally I created [this PR](https://github.com/octokit/octokit.net/pull/2874) to correct some links using it when they shouldn't be). I wonder if readthedocs.io assumes our default branch incorrectly?

**Comment by kfcampbell** [comment]
Closing, conversation will take place in https://github.com/octokit/octokit.net/issues/2863
